src/components/Certify/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcrs-shared-components/certify",
-    "version": "1.0.6",
+    "version": "1.0.7",
```

Comment:
No, you don't do this. Lerna does that in step 14.

https://github.com/bcgov/bcrs-shared-components/blob/master/README.md

You may not have sufficient permissions to do steps 14 and 15. Just ask Cam or me to do it for you.

Reply:
Ah oh gotcha. changed it back